cdap-docs/developers-manual/source/data-exploration/streams.rst

```diff
@@ -135,17 +135,23 @@ For example::
 CSV and TSV Formats
 -------------------
 The ``csv`` (comma separated values) and ``tsv`` (tab separated values) formats read event bodies as delimited text.
-They have two settings, ``charset`` for the text charset, and ``delimiter`` for the delimiter.
+They have three settings, ``charset`` for the text charset, ``delimiter`` for the delimiter, and ``mapping`` for
+column index to schema field mapping.
 The ``charset`` setting defaults to ``utf-8``. The ``delimiter`` setting defaults to a comma
-for the ``csv`` format and to a tab for the ``tsv`` format.
+for the ``csv`` format and to a tab for the ``tsv`` format. The ``mapping`` setting is optional, and
+is in the format ``index0:field0,index1:field1``. If provided, the CSV field order will be decided by the mapping
+rather than using the schema field order. E.g., if the ``mapping`` is ``1:age,0:name``, then the stream event
+``foo,123,82`` will be parsed as ``{"age":123, "name":"foo"}``.
```

Comment:
For example, if the ``mapping`` is ``1:age,0:name``, then the stream event
``foo,35,95409,CA`` will be parsed as ``{"age":35, "name":"foo"}``.